packages/core/src/index.ts

```diff
@@ -3,9 +3,8 @@ import {
   ThemeContext as EmotionContext,
   InterpolationWithTheme,
 } from '@emotion/core'
-// @ts-ignore
 import { css, Theme } from '@theme-ui/css'
-import React from 'react'
+import * as React from 'react'
```

Comment:
See https://github.com/facebook/react/pull/18102

It _might_ help with https://github.com/system-ui/theme-ui/issues/1000. I didn't test it.

Reply:
We should probably consider using star import for React everywhere.